gulpfile.js

```diff
@@ -10,6 +12,12 @@ const paths = {
     foundation_js: [`${foundationDir}/scss/*`,`${foundationDir}/scss/*/**`,]
 };
 
+gulp.task('default', function () {
+    return gulp.src('src/file.txt')
```

Comment:
Should `src/file.txt` be pointing at the `sortable.js` file? I think the path needed will be `ynr/assets/js/sorttable.js`. And also I think the `dest` location will need updating to the be the same?

In practice how does this then get triggered, is it part of the `build` step? Or does it need to be triggered manually?

Reply:
Oh good catch. Yes, it's part of the build step. https://github.com/DemocracyClub/yournextrepresentative/blob/a6e768ec64b23eaede3eba7f021a86e741b70598/package.json#L4